app/app.py

```diff
@@ -14,19 +14,33 @@
 
 def create_app(test_config=None):
 
+    # app = Flask(__name__,
+    #         static_url_path='/static', 
+    #         static_folder='../frontend/static',
+    #         template_folder='../frontend/templates')
+
+
+    # get absolute path of current file
+    base_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
+
+    # construct new paths relative to where the current file is
+    static_folder = os.path.join(base_dir, 'frontend', 'static')
+    template_folder = os.path.join(base_dir, 'frontend', 'templates')
+
+
     app = Flask(__name__,
-            static_url_path='/static', 
-            static_folder='../frontend/static',
-            template_folder='../frontend/templates')
+        static_url_path='/static', 
+        static_folder=static_folder,
+        template_folder=template_folder)
 
     app.json_encoder = CustomJSONEncoder
 
-    app.config.from_file('../config/default.json', json.load)
+    app.config.from_file('/app/config/default.json', json.load)
```

Comment:
Would need to change this here to reference `base_dir` as well so that the local builds don't break.